routes/users.js

```diff
@@ -0,0 +1,51 @@
+const express = require('express');
+const { keysToCamel } = require('../common/utils');
+const { db } = require('../server/db');
+
+const userRouter = express.Router();
+
+userRouter.get('/', async (req, res) => {
+  try {
+    const allUsers = await db.query(`SELECT * FROM users;`);
+    res.status(200).json(keysToCamel(allUsers));
+  } catch (err) {
+    console.log(err);
+    res.status(500).send(err.message);
+  }
+});
+
+
+userRouter.get('/pending-accounts', async (req, res) => {
+  try {
+    const pendingAccounts = await db.query(`SELECT * FROM users WHERE approved = FALSE;`);
+    res.status(200).json(keysToCamel(pendingAccounts));
+  } catch (err) {
+    console.log(err);
+    res.status(500).send(err.message);
+  }
+});
+
+
+userRouter.post('/', async (req, res) => {
+  try {
+    const { id, email, type, approved } = req.params;
+  } catch (err) {
+
+  }
+});
+
+userRouter.put('/:uid', async (req, res) => {
+  try {
+
+  } catch (err) {
+
+  }
+});
+
+userRouter.delete('/:uid', async (req, res) => {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'res' is defined but never used.